src/internal/m365/collection/drive/delta_tree.go

```diff
@@ -450,6 +451,27 @@ func walkTreeAndBuildCollections(
 	return nil
 }
 
+func (face *folderyMcFolderFace) generateExcludeItemIDs() map[string]struct{} {
+	result := map[string]struct{}{}
+
+	for iID, pID := range face.fileIDToParentID {
+		if _, itsAlive := face.folderIDToNode[pID]; !itsAlive {
+			// don't worry about items whose parents are tombstoned.
+			// those will get handled in the delete cascade.
+			continue
+		}
+
+		result[iID+metadata.DataFileSuffix] = struct{}{}
+		result[iID+metadata.MetaFileSuffix] = struct{}{}
+	}
+
+	for iID := range face.deletedFileIDs {
+		result[iID] = struct{}{}
```

Comment:
nit: these should be suffixed as well iirc. They need to match exactly with what kopia will be examining

Reply:
Yup, that should be fixed in a later PR (when I round up all the fix changes).